### DISCUSSION TITLE
WLAN: Automatischer Kanalwechsel in Freetz-ng konfiguriebar?

### DISCUSSION THREAD

**Discussion by Joachim-42he** [Freetz-NG/freetz-ng#370] [Allgemein]
Freetz-Noob-Frage:
Bei mir zu Hause gibt es einfach zu viele WLANs. Obwohl es im 5 GHz-Band auch ziemlich „laut“ ist, muss ich es nutzen, da es Im 2,4 GHz-Band so „laut“ ist, dass es unmöglich ist ein stabiles Netz hinzubekommen.
Leider gibt es eine blöde Kombination:
Viele WLAN-Geräte, die offiziell das 5 GHz-Band nutzen können, können nur die Kanäle 36, 38, 44, 48 nutzen (um Zertifizierungskosten zu vermeiden). Das betrifft bei mir zwei Fire-TV-Sticks und ein Notebook. :-(

Die FritzBox hat ein, eigentlich sinnvolles Feature, dass mir Kopfschmerzen bereitet:
Auch wenn man einen Funkkanal fest vorgibt (zB 44) wechselt die Fritz!Box bei "Lärm" auf dem Kanal automatisch auf einen höheren Kanal (zB 108) und wartet ab, dass es auf dem eingestellten Kanal wieder „leiser“ wird.
Siehe hierzu auch das ["Wissensdokument #3311"](https://avm.de/service/wissensdatenbank/dok/FRITZ-Box-7430/3311_FRITZ-Box-wechselt-den-manuell-festgelegten-Funkkanal/)

Immer wenn das passiert (inzwischen mehrmals am Tag) sehen meine drei Geräte kein aktives WLAN mehr. :-(

Das beste: Laut AVM kann man diese Feature nicht abstellen oder konfigurieren! :-(

Kann man dieses Feature in Freetz-ng konfigurieren oder abstellen?

**Comment by fda77** [comment]
Es ist kein Bug sondern ein Feature ^^ freetz ändert am Wlan nichts.Ich hatte das auch mal und hab es "behoben" indem ich das Gerät zurückgesetzt hab und alle Einstellungen wieder zusammengeklickt hab.
Ansonsten schau doch ob dein Gerät von Openwrt unterstützt wird. Da du 7430 verlinkt hast: https://git.openwrt.org/?p=openwrt/openwrt.git;a=commit;h=6c85a449a988876fdfd4ce84d6aa5ffe4a2820c1

**Reply by Joachim-42he** [reply]
Ich weiß! 😁  Aber dieses Feature ist in meinem Fall extrem lästig.
Ich habe eine 7490.

**Comment by fda77** [comment]
Ich würde zuerst Werksresett versuchen, das half bei mir.
Mir ist übrigens neu dass dies mittlerweile offiziell ein Feature ist! Schau dir mal /var/flash/wlan.cfg an, dort gibt es Dinge wie "autochannel_plus" und "autochannel_plus_11a".
Oder im Terminal mit den iw*  (zb iwconfig) Befehlen, fall es sie auf der 7490 gibt
Aber ich glaub im IPPF können dir mehr Leute helfen, hier schreiben nicht viele

**Reply by Joachim-42he** [reply]
> Mir ist übrigens neu dass dies mittlerweile offiziell ein Feature ist!

Das war zumindest die Antwort vom AVM-Support.

Werksreset werde ich mal versuchen und im Terminal schaue ich mich auch mal um, Danke!
